tests/Test_selenium_unittest.py

```diff
@@ -0,0 +1,177 @@
+        # Login to the site.
+        driver = self.login_page()
+        # Navigate to the engagement page
+        driver.get(self.base_url + "engagement")
+        # Select a previously created engagement title
+        driver.find_element_by_partial_link_text("Quick Security Testing").click()
+        # "Click" the dropdown button to see options
+        driver.find_element_by_id("dropdownMenu1").click()
+        # "Click" the Edit Test option
+        driver.find_element_by_link_text("Add Notes").click()
+        # Select entry, clear field and input note
+        driver.find_element_by_id("id_entry").clear()
+        driver.find_element_by_id("id_entry").send_keys("This is a sample note for all to see.")
+        # "Click" the submit button to complete the transaction
+        driver.find_element_by_xpath("//input[@value='Add Note']").click()
+        # Query the site to determine if the Test has been updated
+        productTxt = driver.find_element_by_tag_name("BODY").text
+        # Assert ot the query to dtermine status of failure
+        self.assertTrue(re.search(r'Note added successfully.', productTxt))
+
+    def test_delete_test(self):
+        # Login to the site. Password will have to be modified
+        # to match an admin password in your own container
+        driver = self.login_page()
+        # Navigate to the engagement page
+        driver.get(self.base_url + "engagement")
+        # Select a previously created engagement title
+        driver.find_element_by_partial_link_text("Quick Security Testing").click()
+        # "Click" the dropdown button to see options
+        driver.find_element_by_id("dropdownMenu1").click()
+        # "Click" the Edit Test option
+        driver.find_element_by_link_text("Delete Test").click()
+        # Type test name into Title field before clicking Delet button
+        driver.find_element_by_id("id_title").clear()  # always clear for inputting
+        driver.find_element_by_id("id_title").send_keys("Quick Security Testing")
+        # "Click" the delete button to complete the transaction
+        driver.find_element_by_css_selector("button.btn.btn-danger").click()
+        # Query the site to determine if the product has been added
+        productTxt = driver.find_element_by_tag_name("BODY").text
+        # Assert ot the query to dtermine status of failure
+        self.assertTrue(re.search(r'Test and relationships removed.', productTxt))
+
+    def tearDown(self):
+        self.driver.quit()
+        self.assertEqual([], self.verificationErrors)
+
+
+def suite():
+    suite = unittest.TestSuite()
+    # Add each test the the suite to be run
+    # success and failure is output by the test
+    suite.addTest(product_unit_test.ProductTest('test_create_product'))
+    suite.addTest(TestUnitTest('test_create_test'))
+    suite.addTest(TestUnitTest('test_edit_test'))
+    suite.addTest(TestUnitTest('test_add_note'))
+    # suite.addTest(TestUnitTest('test_delete_test'))
```

Comment:
Uncomment this for the real PR. Also be sure to clean up any extra resources created. Deleting products also deletes any engagements/findings/.etc.

Reply:
product test file completed

Reply:
yea.. i forgot to uncomment that. Done so now.